### PR TITLE
Update info about Eclipse Corrosion

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

File: docs/user/manual.adoc [hunk 1]
```diff
@@ -384,11 +384,9 @@ If the LSP binary is not available, GNOME Builder can install it when opening a
 
 === Eclipse IDE
 
-Prerequisites: You have installed the <<rust-analyzer-language-server-binary,`rust-analyzer` binary>>.
-
 Support for Rust development in the Eclipse IDE is provided by link:https://github.com/eclipse/corrosion[Eclipse Corrosion].
-While it currently uses RLS as default, you can successfully configure it so the IDE will use `rust-analyzer` instead.
-To do so, with an Eclipse IDE where Corrosion is installed, just go to __Window > Preferences > Rust__ and edit the __Path to Rust Language Server__ entry to reference the path to `rust-analyzer`.
+If available in PATH or in some standard location, `rust-analyzer` is detected and powers edition of Rust files without further configuration.
```
Comment (matklad) [inline docs/user/manual.adoc:388]:
```suggestion
If available in PATH or in some standard location, `rust-analyzer` is detected and powers editing of Rust files without further configuration.
```

?

Comment (mickaelistria) [reply docs/user/manual.adoc:388]: Good to me. Should I use the "commit suggestion" capability or do you prefer a different workflow?

**Comment by lnicola** [comment]
changelog fix

**Pull request merged**